benchmarks/tables/fh-table.jl

```diff
@@ -13,9 +13,9 @@ uvar = fill(Inf, 5)
 data, simulate, resid, misfit, x0 = RegularizedProblems.FH_smooth_term()
 model = ADNLPModel(misfit, ones(5), lvar, uvar)
 f = LBFGSModel(model)
-λ = cstr ? 2.0e1 : 1.0e1
+λ = cstr ? 4.0e1 : 1.0e1
```

Comment:
Could you explain why you changed these values here and in `input_struct.jl`?

Reply:
It is empirical, I realized I had better results with these.